build.gradle

```diff
@@ -124,7 +124,7 @@ jacocoTestReport {
 
 test {
   testLogging {
-    showStandardStreams = true
+    events("passed", "skipped", "failed")
```

Comment:
I liked the way it way with showStandardStreams = true

it spamed the log but it finally printed needed information and if u know where to search for the right info u will find it

Reply:
I can live with it if we can suppress it for successful tests

Reply:
Showing standard streams only for failure can be done it seems but is very ugly and not worth it imo:
https://stackoverflow.com/questions/36558340/during-gradle-test-show-standard-out-and-err-only-for-failed-tests 

@axel1200 I kinda agree with Brutus, searching though Travis output is rather annoying currently, and since most Exceptions will now actually fail our tests and therefore get printed i think its fine to disable standard streams by default.

Reply:
I dont have a problem with it... I search for FAILED and there are 8 ones i find i skip the first 5 till i get to the failed test

Reply:
I removed it for now from the PR. But I see this differently. The logs are garbage with shitload of exceptions flying for passing tests.

Reply:
They are even worse if the Exception is not in them :D

Reply:
Reading the exceptions of for example half-broken chat test over and over does not help in everyday work. If someone wants to fix them, he can just switch `showStandardStreams` to true locally. And we should probably make an issue for that. And somebody should probably go and fix it.